query_server/spi/src/query/logical_planner.rs

```diff
@@ -126,6 +134,21 @@ pub struct CreateDatabase {
     pub options: DatabaseOptions,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct DescribeDatabase {
+    pub database_name: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct DescribeTable {
+    pub table_name: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ShowTable {
+    pub database_name: String,
```

Comment:
Would it be better to use option? Because the user may execute `show tables` without database name.

Reply:
if empty, it will get the current database.

Reply:
Use Option::None to represent the database is not specified, which will make the logic easier to understand

Reply:
same effect, i don't think there any difference between these two way